app/Http/Controllers/Api/AssetsController.php

```diff
@@ -189,6 +190,10 @@ public function index(Request $request, $audit = null)
             $assets->ByDepreciationId($request->input('depreciation_id'));
         }
 
+        if ($request->filled('byod')) {
+            $assets->ByDepreciationId($request->input('byod'));
```

Comment:
This definitely doesn't look right to me.

Reply:
Derp - you're right - copy pasta :( 

![Demote_Copypasta](https://user-images.githubusercontent.com/197404/213323122-b99b9f9f-910e-4984-8307-92fb96746a61.jpg)

Fixed.